client/app/reducers/homeReducer.js

```diff
@@ -1,7 +1,7 @@
 var actions = require('../actions/actions');
 
 module.exports = function(state, action) {
-  var initialState = {quote: null};
+  var initialState = {quote: {quote: 'Please check back for a new quote.', author: 'oMdo Dev Team'}};
   state = state || initialState;
```

Comment:
This does allow the homepage to render. However, without the cron job or a set of default quotes, the 'Please check back...' will always show up.